test/model/account/Account.spec.ts

```diff
@@ -45,4 +46,14 @@ describe('Account', () => {
         expect(account.address).to.not.be.equal(undefined);
     });
 
+    it('signData', () => {
+        const account = Account.createFromPrivateKey(
+            'AB860ED1FE7C91C02F79C02225DAC708D7BD13369877C1F59E678CC587658C47',
+            NetworkType.MIJIN_TEST,
+        );
+        const publicAccount = account.publicAccount;
+        const signed = account.signData('catapult rocks!');
+        expect(publicAccount.verifySignature('catapult rocks!', signed))
+            .to.be.equal(true);
```

Comment:
`to.be.true`

I've seen this repeats in other tests as well ^^

Reply:
`to.be.true` is an issue with tslint, we should 

a) change the tslint rules
b) use `equal(true)`.

I like `to.be.true`

Reply:
I prefer to accept `to.be.true` as well, but np it's a detail